packages/contracts/contracts/voting/majority/MajorityVotingBase.sol

```diff
@@ -225,24 +311,33 @@ abstract contract MajorityVotingBase is
         return votingSettings.minParticipation;
     }
 
-    /// @inheritdoc IMajorityVoting
+    /// @notice Returns the minimum duration parameter stored in the voting settings.
+    /// @return The minimum duration parameter.
     function minDuration() public view virtual returns (uint64) {
         return votingSettings.minDuration;
     }
 
-    /// @inheritdoc IMajorityVoting
+    /// @notice Returns the minimum voting power required to create a proposa stored in the voting settings.
+    /// @return The minimum voting power required to create a proposal.
     function minProposerVotingPower() public view virtual returns (uint256) {
         return votingSettings.minProposerVotingPower;
     }
 
-    /// @inheritdoc IMajorityVoting
+    /// @notice Returns the vote mode stored in the voting settings.
+    /// @return The vote mode parameter.
     function votingMode() public view virtual returns (VotingMode) {
         return votingSettings.votingMode;
     }
 
-    /// @inheritdoc IMajorityVoting
+    /// @notice Returns all information for a proposal vote by its ID.
+    /// @param _proposalId The ID of the proposal.
+    /// @return open Wheter the proposal is open or not.
+    /// @return executed Wheter the proposal is executed or not.
+    /// @return parameters The parameters of the proposal vote.
+    /// @return tally The current tally of the proposal vote.
+    /// @return actions The actions to be executed in the associated DAO after the proposal has passed.
     function getProposal(uint256 _proposalId)
-        public
+        external
```

Comment:
better to be public if inherriting contract needs proposal object as proposals array is private or will be soon.

Reply:
Changed in 6d65fb6f16db1f60b3fbe109d4842b7296e9762e